tests/feature/ExportDumpTest.php

```diff
@@ -18,14 +19,48 @@ class ExportDumpTest extends TestCase
     protected string $filePath;
     protected string $emptyDumpPath;
 
+    const POSTGRES_CREATE = '--create';
+    const POSTGRES_CLEAN = '--clean';
+    const POSTGRES_VERBOSE = '--verbose';
+    const POSTGRES_SCHEMA_ONLY = '--schema-only';
+    const NO_TABLESPACES = '--no-tablespaces';
+    const MYSQL_SET_GTID_PURGED = '--set-gtid-purged=OFF';
+    const MYSQL_NO_CREATE_DB = '--no-create-db';
+    const MYSQL_SKIP_COMMENTS = '--skip-comments';
+    const MYSQL_SKIP_SET_CHARSET = '--skip-set-charset';
+    const MYSQL_NO_DATA = '--no-data';
+    const PROTECTOR_WITH_CREATE_DB = 'withCreateDb';
+    const PROTECTOR_WITH_DROP_DB = 'withDropDb';
+    const PROTECTOR_WITH_COMMENTS = 'withComments';
+    const PROTECTOR_WITH_CHARSETS = 'withCharsets';
+    const PROTECTOR_WITH_DATA = 'withData';
+    const PROTECTOR_WITH_TABLESPACES = 'withTablespaces';
+    const PROTECTOR_CONFIG_BASELINE = [
+        'pgsql' => [
+            self::POSTGRES_CREATE => false,
+            self::POSTGRES_CLEAN => false,
+            self::POSTGRES_VERBOSE => false,
+            self::POSTGRES_SCHEMA_ONLY => false,
+            self::NO_TABLESPACES => true,
```

Comment:
so does postgres have tablespaces or is this to ensure this is not set?

Reply:
and should this be on for postgres?

Reply:
The only reason we disabled tablespaces on MySQL is that a high privilege is needed for this. As long as we don't encounter any issues with dumping tablespaces on PostgreSQL, I see no reason why we should disable tablespaces there? 🤔